packages/tlschema/src/assets/TLBookmarkAsset.ts

```diff
@@ -14,7 +14,6 @@ export type TLBookmarkAsset = TLBaseAsset<
 	}
 >
 
-// --- VALIDATION ---
```

Comment:
There are a lot of these comments that I'm pulling out. We can have guides outside of the code.